firestore-bigquery-export/scripts/gen-schema-view/src/schema.ts

```diff
@@ -576,7 +576,10 @@ export function subSelectQuery(query: string, filter?: string[]): string {
 }
 
 function qualifyFieldName(prefix: string[], name: string): string {
-  return prefix.concat(name).join("_");
+  const notAlphaDigitUnderscore = /([^a-zA-Z0-9_])/;
```

Comment:
```suggestion
  const notAlphaDigitUnderscore = /([^a-zA-Z0-9_])/g;
```

I believe this regex needs the global modifier - otherwise it will only replace the first character it comes across and leaving the rest

Reply:
Nit: can we call this `notAlphanumericUnderscore`?